api/classes/publisher.js

```diff
@@ -310,6 +318,23 @@ class BasePublisher {
     return publishedFilesQueryBuilder
     .getAllModifiedFiles(this.publishedProject.id)
     .then(function(publishedFiles) {
```

Comment:
Thanks for the feedback @gideonthomas.  I've made the changes and tested, it's working well on my end.  Please let me know if there are any issues